src/main/java/frc/robot/Robot.java

```diff
@@ -54,7 +51,10 @@ public void robotPeriodic() {
         // commands, running already-scheduled commands, removing finished or interrupted commands,
         // and running subsystem periodic() methods.  This must be called from the robot's periodic
         // block in order for anything in the Command-based framework to work.
-        robotContainer.updateShuffleBoard();
+
+        // TODO: why is this here?
+        // robotContainer.updateShuffleBoard();
+
```

Comment:
This makes things like the Shuffleboard field periodically update. We need this for it to work.

Reply:
updateShuffleboard() is a poor idea.

Shuffleboard should not be controlling updates; Shuffleboard should have no knowledge of which systems are active; the subsystems should be submitting updates to Shuffleboard.

In any event, robotPeriodic has the call now, so this comment should not hold up the PR.